docs/index.rst

```diff
@@ -51,7 +51,7 @@ Installation
 Requirements:
 
 * Python 2.7+ or 3.4+
-* `Django`_ 1.6 through 1.9
+* `Django`_ 1.8 through 1.9
```

Comment:
We're not testing on Django 1.10 yet, but will be in my next PR (along with 1.11 and django master, though likely those two marked as "allowed failures" on Travis initially).